static/js/components/ObservationPlanRequestForm.jsx

```diff
@@ -47,6 +51,111 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
+const ObservationPlanGlobe = ({ loc, instrument, setSelectedFields }) => {
+  const dispatch = useDispatch();
+
+  const displayOptions = [
+    "localization",
+    "sources",
+    "galaxies",
+    "instrument",
+    "observations",
+  ];
+  const displayOptionsDefault = Object.fromEntries(
+    displayOptions.map((x) => [x, false])
+  );
+  displayOptionsDefault.localization = true;
+  displayOptionsDefault.instrument = true;
+
+  const [skymapInstrument, setSkymapInstrument] = useState(null);
+  useEffect(() => {
+    const fetchSkymapInstrument = async () => {
+      const response = await dispatch(
+        GET(
+          `/api/instrument/${instrument.id}?includeGeoJSONSummary=True&localizationDateobs=${loc.dateobs}&localizationName=${loc.localization_name}`,
+          "skyportal/FETCH_INSTRUMENT"
+        )
+      );
+      setSkymapInstrument(response.data);
+    };
+    fetchSkymapInstrument();
+  }, [dispatch, setSkymapInstrument, loc, instrument]);
+
+  const handleAddObservationPlanFields = async (obsPlanSkymapInstrument) => {
+    const selectedFields = obsPlanSkymapInstrument?.fields?.filter(
+      (f) => f?.selected
+    );
+    const selectedIds = selectedFields.map((f) => f?.field_id);
+    setSelectedFields(selectedIds);
+  };
+
+  return (
+    <div>
+      {!loc ? (
+        <div>
+          <CircularProgress />
+        </div>
+      ) : (
+        <div>
+          <LocalizationPlot
+            loc={loc}
+            instrument={skymapInstrument}
+            options={displayOptionsDefault}
+            height={300}
+            width={300}
```

Comment:
why so small? I like the size of the one in GcnSelectionForm.

Reply:
Agreed. And honestly, even on the gcn selection form it could be bigger or the skymap could use an initial zoom. It is very small compared to the huge part of the page dedicated to that form. There's a bunch of unused empty space.